.env.example

```diff
@@ -7,15 +7,19 @@ REDIS_URL=redis://127.0.0.1:6379
 # Configure the RPC endpoints for each blockchain (deployment) supported in deployment.service.ts (required)
 # If the database is empty, these endpoints will be used to query historical data, ideally a paid plan should be used so you won't be rate limited.
 ETHEREUM_RPC_ENDPOINT=xxxxxxxxxx
+HEDERA_RPC_ENDPOINT=xxxxxxxxxx
 SEI_RPC_ENDPOINT=xxxxxxxxxx
 CELO_RPC_ENDPOINT=xxxxxxxxxx
 BLAST_RPC_ENDPOINT=xxxxxxxxxx
 
 # Configure the API keys for the external services
 # Coingecko: if under free tier, changes to api url and auth header are required. Required if deployment to Ethereum network is supported.
-COINGECKO_API_KEY=CG-xxxxxxxxxx 
+COINGECKO_API_KEY=CG-xxxxxxxxxx
+COINGECKO_API_URL=https://api.coingecko.com/api/v3
```

Comment:
if under free tier, changes to api url and auth header are required, like below:
```
COINGECKO_API_URL=https://api.coingecko.com/api/v3
COINGECKO_API_AUTH_HEADER=x-cg-demo-api-key
```